src/components/Modals/NewCourse/SelectedRequirementEditor.vue

```diff
@@ -1,102 +1,115 @@
 <template>
   <div>
-    <div v-if="chosenRequirementText.length > 0">
+    <div v-if="requirementsThatAllowDoubleCounting.length > 0">
       <div class="newCourse-title">
-        You will let this class fulfill the following requirements(s):
+        This class automatically fulfills the following requirement(s):
       </div>
-      <div class="newCourse-requirements-container">
-        <div class="newCourse-requirements">
-          {{ chosenRequirementText }}
-        </div>
-      </div>
-    </div>
-    <div v-if="requirementsThatAllowDoubleCounting.length > 0">
-      <div class="newCourse-title">This class can auto-fulfill the following requirement(s):</div>
       <div class="newCourse-requirements-container">
         <div class="newCourse-requirements">
           {{ requirementsThatAllowDoubleCounting.join(', ') }}
         </div>
       </div>
     </div>
-    <div v-if="relatedRequirements.length > 0">
+    <div v-else class="newCourse-requirements-container newCourse-requirements">
+      This class does not automatically fulfill any requirements.
+    </div>
+    <div v-if="nonAutoRequirements.length > 0">
       <div class="newCourse-title">This class can fulfill the following requirement(s):</div>
-      <div v-if="!editMode" class="newCourse-requirements-container">
-        <div class="newCourse-requirements">
-          {{ relatedRequirements.map(it => it.name).join(', ') }}
+      <div v-if="!editMode">
+        <div class="newCourse-requirements-container">
+          <div class="newCourse-title">
+            <span v-for="(reqName, index) in nonAutoRequirementsTextArray" :key="index">
+              <strong v-if="reqName.selected" class="newCourse-name">{{
+                index === nonAutoRequirementsTextArray.length - 1
+                  ? reqName.name
+                  : reqName.name + ', '
+              }}</strong>
+              <span v-else>{{
+                index === nonAutoRequirementsTextArray.length - 1
+                  ? reqName.name
+                  : reqName.name + ', '
+              }}</span>
+            </span>
```

Comment:
@hahnbeelee I replaced `nonAutoRequirementsText` with `nonAutoRequirementsTextArray` which is an array of objects. Each object has the name of the requirement and a boolean for whether or not it was selected (and should have the strong tag). Using those fields, I conditionally apply the strong tag or span tag. I take a look at the index of `reqName` in `nonAutoRequirementsTextArray` to see if we need to have a `', '` or not.

Reply:
Looks good to me!